src/main/java/org/elasticsearch/action/admin/indices/analyze/AnalyzeRequest.java

```diff
@@ -36,102 +37,56 @@
  */
 public class AnalyzeRequest extends SingleCustomOperationRequest<AnalyzeRequest> {
 
-    private String text;
-
-    private String analyzer;
-
-    private String tokenizer;
-
-    private String[] tokenFilters = Strings.EMPTY_ARRAY;
-
-    private String[] charFilters = Strings.EMPTY_ARRAY;
-
-    private String field;
+    private BytesReference source;
+    private boolean unsafe;
```

Comment:
Nevermind, I see why it is "unsafe" now. But do we really need to have all these crazy options on the request? Why can't we just always copy?